wp-includes/sqlite/class-wp-sqlite-translator.php

```diff
@@ -3570,9 +3570,7 @@ private function get_key_definitions( $table_name, $columns ) {
 			$key_definition[] = 'KEY';
 
 			// Remove the prefix from the index name if there is any. We use __ as a separator.
-			$index_name = strstr( $key['index']['name'], '__' )
-				? explode( '__', $key['index']['name'] )[1]
-				: $key['index']['name'];
+			$index_name = explode( '__', $key['index']['name'], 2 )[1] ?? $key['index']['name'];
```

Comment:
This looks good to me, but it uses a null coalescing operator which is not supported by our declared minimum PHP version of 5.6. We already have a number of uses of the null coalescing operator, so it's likely we just need to bump the minimum PHP version.

https://github.com/WordPress/sqlite-database-integration/blob/5a2576ba82053c42c9388702727e049a2ccf1eb0/readme.txt#L6

@aristath what do you think?

Reply:
I went by the PHP version specified in [composer.php](https://github.com/WordPress/sqlite-database-integration/blob/5a2576ba82053c42c9388702727e049a2ccf1eb0/composer.json#L11) and also found other instances of code that wouldn't be compatible with 5.6. I agree that we should bump the version in `readme.txt` so that it matches the version defined in `composer.php`.

Reply:
You're right, we just forgot to change the minimum required PHP version in the readme file 😅 
I pushed a commit in the `develop` branch already, upping the version 👍

Reply:
Thank you both!